packages/renderer/src/lib/chat/components/sidebar-history/history.svelte

```diff
@@ -171,4 +202,19 @@ async function handleDeleteChat(): Promise<void> {
 			</AlertDialogFooter>
 		</AlertDialogContent>
 	</AlertDialog>
+  <AlertDialog bind:open={alertAllDialogOpen}>
+    <AlertDialogContent>
+      <AlertDialogHeader>
+        <AlertDialogTitle>Delete all conversations?</AlertDialogTitle>
+        <AlertDialogDescription>
+          This action cannot be undone. This will permanently delete all of your chats.
+        </AlertDialogDescription>
+      </AlertDialogHeader>
+      <AlertDialogFooter>
+        <AlertDialogCancel>Cancel</AlertDialogCancel>
+        <AlertDialogAction onclick={handleDeleteAllChats}>Continue</AlertDialogAction>
+      </AlertDialogFooter>
+    </AlertDialogContent>
+  </AlertDialog>
```

Comment:
suggestion (not blocking) there is another issue to use Kortex/Podman Desktop ui dialog rather than these one